.github/workflows/ci-cd.yml

```
@@ -30,97 +30,54 @@ jobs:
+          --health-retries=5
+    env:
+      DATABASE_URL: postgres://local_user:local_password@localhost:5432/local_db
+      PORT: 3001
     steps:
       - uses: actions/checkout@v3
+
       - uses: pnpm/action-setup@v2
         with:
           version: 10.6.3
+
       - uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)